packages/patternfly-4/react-core/src/components/List/List.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/List/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Anything that can be rendered inside of the list */
+  children: PropTypes.node,
+  /** Additional classes added to the list. */
+  className: PropTypes.string,
+  /** Flag to show the inline list modifier class. */
+  isInline: PropTypes.bool,
+  /** Flag to show the grid list modifier class. */
+  isGrid: PropTypes.bool
+};
+
+const defaultProps = {
+  children: null,
+  className: '',
+  isInline: false,
+  isGrid: false
+};
+
+const List = ({ className, children, isInline, isGrid, ...props }) => (
+  <ul
+    {...props}
+    className={css(styles.list, isInline && styles.modifiers.inline, isGrid && styles.modifiers.grid, className)}
```

Comment:
We should use a variant her for grid vs inline.  I assume it can be one or the other and not both at the same time.  Look at how Button is doing it with ButtonVariant.  We should double check with design on this.

Reply:
There is no current design for list but I spoke with @andybraren and you're correct in your assumption that it will be either one or the other so I'm fine with changing this to use variant.